lib/memory_profiler/results.rb

```diff
@@ -97,6 +97,12 @@ def pretty_print(io = STDOUT, options = {})
       nil
     end
 
+    def print_to_file(file = 'memory_profiler.txt', options = {})
```

Comment:
if we are to include this ... should be named pretty_print_to_file ... I am not 100% convinced we need it though. 

Maybe just change `pretty_print` to take in `io_or_filename`